### ISSUE TITLE
Give back "Hide service bar labels" option for make icon-only Service bar

### ISSUE THREAD

**Issue by MurzNN** [TheGoddessInari/hamsket#139]
I have use Rambox with left Service bar and use "Hide service bar labels" option for make it width minimal: 
![image](https://user-images.githubusercontent.com/336662/67842972-35a9a380-fb0c-11e9-8701-f545cbe8daa7.png)
But in Hamsket this option is disappear! Can you please give it back, or describe why you remove it?

**Comment by MurzNN** [comment]
Here is Rambox options:
![изображение](https://user-images.githubusercontent.com/336662/68081295-e372c600-fe1c-11e9-9424-b6e206662aba.png)
And here is Hamsket:
![изображение](https://user-images.githubusercontent.com/336662/68081319-32206000-fe1d-11e9-8d83-45f1fd73431b.png)

**Comment by MurzNN** [comment]
Or maybe this feature is not ported from Rambox source? https://github.com/ramboxapp/community-edition/issues/2101 is issue about this feature, and https://github.com/bertyhell/community-edition/commit/d726ed0b2362213ad696af9d3f2c1436f894bd94 is commit.

**Comment by TheGoddessInari** [comment]
Yeah, I'm not porting **everything** from Rambox CE. I'll look into this this week, however.

Sorry about delays, was in the hospital for a while, but actively checking on Hamsket stuff now.
